lib/spack/spack/build_systems/python.py

```diff
@@ -97,10 +97,11 @@ def configure(self, spec, prefix):
 
     extends('python')
 
-    def setup_file(self, spec, prefix):
+    def setup_file(self):
```

Comment:
Because it's not a phase... but I would have pointed out the change before tagging this ready (if you didn't precede me to it) :blush: